internal/wrappers/application-http.go

```diff
@@ -0,0 +1,54 @@
+package wrappers
+
+import (
+	"encoding/json"
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+	"net/http"
+)
+
+type ApplicationsHTTPWrapper struct {
+	path string
+}
+
+func NewApplicationsHTTPWrapper(path string) ApplicationsWrapper {
+	return &ApplicationsHTTPWrapper{
+		path: path,
+	}
+}
+
+func (a *ApplicationsHTTPWrapper) Get(params map[string]string) (*ApplicationsResponseModel, *ErrorModel, error) {
+	if _, ok := params[limit]; !ok {
+		params[limit] = limitValue
```

Comment:
where is the limit value come from? it's get by name no?

Reply:
That sounds good, it's hard to understand it if not. but maybe we don't need it here, no?